gulp/ts.js

```diff
@@ -7,7 +7,7 @@ var ts = require('gulp-typescript');
 var tslint = require('gulp-tslint');
 
 gulp.task('ts-build', function () {
-    return gulp.src(['at-*.ts', 'ts/**/*.d.ts', 'test/**/*.ts']).pipe(ts({
+    return gulp.src(['at-*.ts', '*.d.ts', 'typings/**/*.d.ts', 'test/**/*.ts']).pipe(ts({
```

Comment:
Not sure how you wanted to include `extend.d.ts` in the gulp build process now that it's out of `ts` and at the project root. I opted for a wildcard to grab all definition files at the root, but we could change this if you have something better in mind.